preup/kickstart/cli.py

```diff
@@ -12,12 +12,20 @@ def __init__(self, args=None):
 
         #self.parser.usage = "%%prog [-v] <content_file>"
 
-        #self.add_args()
+        self.add_args()
         if args:
             self.opts, self.args = self.parser.parse_args(args=args)
         else:
             self.opts, self.args = self.parser.parse_args()
 
+    def add_args(self):
+        self.parser.add_option(
+            "--force",
```

Comment:
I don't think that force is the best name here. I would use the same as at least yum/dnf has, i.e. -y option, which makes all the questions to be answered yes automatically.